README.md

```diff
@@ -1,5 +1,71 @@
-# matchminerCurate
-This application was generated using JHipster 4.13.3, you can find documentation and help at [http://www.jhipster.tech/documentation-archive/v4.13.3](http://www.jhipster.tech/documentation-archive/v4.13.3).
+# MatchMinerCurate
+MatchMinerCurate is a realtime trial curation platform by using [Clinical Trial Markup Language] (CTML) to label 
```

Comment:
Can we add a link out to the markup language?

Reply:
I already add the link on line 201.